README.md

```diff
@@ -80,7 +80,7 @@ Introduction - [slides](http://dask.github.io/dask-tutorial/introduction.html)
 
 4.  Imperative Programming
 
-    *  [Imperative - `do`](04-Imperative.ipynb)
```

Comment:
I think imperative is fine personally :) 

Maybe better to clarify with and example/Wikipedia link rather than terminology change?